database/src/main/kotlin/no/nav/su/se/bakover/database/behandling/BehandlingRepo.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.su.se.bakover.database.behandling
+
+import no.nav.su.se.bakover.common.UUID30
+import no.nav.su.se.bakover.domain.Attestant
+import no.nav.su.se.bakover.domain.Behandling
+import no.nav.su.se.bakover.domain.Saksbehandler
+import no.nav.su.se.bakover.domain.behandling.Behandlingsinformasjon
+import java.util.UUID
+
+interface BehandlingRepo {
+    fun hentBehandling(behandlingId: UUID): Behandling?
+    fun oppdaterBehandlingsinformasjon(behandlingId: UUID, behandlingsinformasjon: Behandlingsinformasjon): Behandling
```

Comment:
Jeg stemmer for at der vi inserter/oppdaterer (side-effekt), bruker Unit som return. Det kan være forvirrende hvis man tror man får tilbake et annet objekt.

Reply:
Dette er noe som er relatert til en større diskusjon rundt "hvordan skal vi nå bruke domene-objektene våre" tenker jeg. Kaller vi alltid innom domene-objektene, eller vil vi i noen tilfeller bare hoppe bukk over domenet og lagre ting i basen? Vi må skaffe oss en felles måte å håndtere dette på slik at vi alltid er confidantè på om objektet vi har i minnet er oppdatert, eller om vi må til basen for å hente oppdatert versjon. Jeg heller uansett mot at vi enten returnerer en oppfrisket versjon av det vi lagret, eller Unit.